backend/domain/Room.js

```diff
@@ -144,12 +145,12 @@ class Room {
     }
 
     /**
-     * Generate a random alphanumeric id
-     * @returns {string} a random alphanumeric id
+     * Generate a univerally unique identifier
+     * @returns {string} a univerally unique identifier
      */
-    // eslint-disable-next-line class-methods-use-this
+    /* eslint class-methods-use-this: ["error", { "exceptMethods": ["getRandomID"] }] */
```

Comment:
This is mainly just for my understanding, but could you please explain what this line does?